tests/Operation/RenameCollectionFunctionalTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Model\CollectionInfo;
+use MongoDB\Model\CollectionInfoIterator;
+use MongoDB\Operation\InsertOne;
+use MongoDB\Operation\ListCollections;
+use MongoDB\Operation\RenameCollection;
+use MongoDB\Tests\CommandObserver;
+use function sprintf;
+use function version_compare;
+
+class RenameCollectionFunctionalTest extends FunctionalTestCase
+{
+    public function testDefaultWriteConcernIsOmitted()
+    {
+        (new CommandObserver())->observe(
+            function () {
+                $operation = new RenameCollection(
+                    'admin',
+                    $this->getCollectionName(),
+                    $this->getCollectionName() . '.renamed',
+                    ['writeConcern' => $this->createDefaultWriteConcern()]
+                );
+
+                $operation->execute($this->getPrimaryServer());
+            },
+            function (array $event) {
+                $this->assertObjectNotHasAttribute('writeConcern', $event['started']->getCommand());
+            }
+        );
+    }
+
+    public function testRenameExistingCollection()
+    {
+        $that = $this;
+        $renamedCollectionName = $this->getCollectionName() . '.renamed';
+        $server = $this->getPrimaryServer();
+
+        $insertOne = new InsertOne('admin', $this->getCollectionName(), ['x' => 1]);
+        $writeResult = $insertOne->execute($server);
+        $this->assertEquals(1, $writeResult->getInsertedCount());
+
+        $operation = new RenameCollection('admin', $this->getCollectionName(), $renamedCollectionName);
+        $commandResult = $operation->execute($server);
+
+        $this->assertCommandSucceeded($commandResult);
```

Comment:
The tests aren’t passing now because the assertion in this line fails; I printed the `$commandResult` object for the error message but am not sure what’s wrong still.

![Screen Shot 2021-07-07 at 4 46 25 PM](https://user-images.githubusercontent.com/46271307/124827153-b5c01780-df43-11eb-93a0-ad9f6473b419.png)